coalib/bears/Bear.py

```diff
@@ -130,6 +130,12 @@ class Bear(Printer, LogPrinterMixin):
     >>> class SomeBear(Bear): pass
     >>> SomeBear.source_location
     '...Bear.py'
+
+    Every lint bear makes use of an executable tool for its operations.
```

Comment:
also I would say

Most linter bears wrap an executable tool or something like that :smile: 

but you can keep this if you want to :+1: